libs/ui-toolkit/slide-toggle/src/lib/slide-toggle.stories.ts

```diff
@@ -0,0 +1,74 @@
+import { storiesOf, moduleMetadata } from '@storybook/angular';
+import { CommonModule } from '@angular/common';
+import { SlideToggleComponent } from './slide-toggle.component';
+import { SlideToggleGroupComponent } from './slide-toggle-group.component';
+import { ToggleData } from './slide-toggle.model';
+import { text, withKnobs, boolean } from '@storybook/addon-knobs';
+
+storiesOf('slide-toggle', module)
+  .addDecorator(
+    moduleMetadata({
+        imports: [CommonModule],
+        declarations: [SlideToggleComponent, SlideToggleGroupComponent],
+ })
+)
+.addDecorator(withKnobs)
+.add(
+    'three different cases about slide-toggle',
```

Comment:
Summary should be descriptive of functionality being demonstrated - in this case, color configurations